cirq/ops/linear_combinations.py

```diff
@@ -342,8 +342,10 @@ def copy(self) -> 'PauliSum':
         factory = type(self)
         return factory(self._linear_dict.copy())
 
-    def expectation_from_wavefunction(self, state: np.ndarray,
-                                      qubit_map: Mapping[raw_types.Qid, int]
+    def expectation_from_wavefunction(self,
+                                      state: np.ndarray,
+                                      qubit_map: Mapping[raw_types.Qid, int],
+                                      check_preconditions: bool = True
```

Comment:
Force keyword arg on `check_preconditions`. Add `check_preconditions` to the `Args:` list.